README.md

```diff
@@ -1 +1,85 @@
 # draft-store-client
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/cmc-citizen-frontend.svg)](https://greenkeeper.io/)
+
+[![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
```

Comment:
@geek-soft Damian, do we want to use standardjs for new Typescript projects/libs?

Reply:
We are happy with standardjs, makes sense to use it for all the projects. There is standardjs rule set for ts-lint (we are using it).